### PR TITLE
implements getPermissionsForRole based

### PR DESCRIPTION
**Pull request by heat**

### CODE REVIEW

**Comment by leleuj** [comment]
The build is broken due to 1 PWD violation: can you fix it?

File: src/main/java/org/pac4j/play/deadbolt2/Pac4jRoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import be.objectify.deadbolt.java.models.Permission;
+import play.libs.concurrent.HttpExecutionContext;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public interface Pac4jRoleHandler {
```
Comment (leleuj) [inline src/main/java/org/pac4j/play/deadbolt2/Pac4jRoleHandler.java]: Can you add some Javadoc with the `@author` and `@since` annotations?

**Comment by leleuj** [comment]
So now we must bind a `Pac4jRoleHandler` to use Deadbolt: I think it would be a good idea to provide a "no operation" default one, wouldn't it?

**Comment by heat** [comment]
Yeph I will fix when going back from cristhimas. 
Given a default implementation that return a empty list we keep same  behavior that we had before the handler.  I think it's a good idea

File: src/main/java/org/pac4j/play/deadbolt2/Pac4jRoleHandler.java (hunk 1 above)
Comment (heat) [inline src/main/java/org/pac4j/play/deadbolt2/Pac4jRoleHandler.java]: added. I marked `since` with *2.6.2*

**Comment by heat** [comment]
All green now.

**Comment by leleuj** [comment]
Thanks

**Pull request merged**